.gitignore

```diff
@@ -10,6 +10,12 @@
 # flatc generated files.
 /node/src/fbs
 
+## Node WorkerChannel addon.
+/node/workerChannel/node_modules
+/node/workerChannel/build
+/node/workerChannel/lib
```

Comment:
Shouldn't we include this `lib` folder in "files" in main `package.json`? Or should it be managed by the `package.json` of the addon folder? What I mean is that:

- We only include JS transpiled files in the mediasoup NPM package and not TS files (AKA `node/src` folder).
- Is the addon properly built if we install mediasoup as a git+ssh dependency in the package.json of whatever app using mediasoup?

Reply:
Reminder

Reply:
Done and tested. package.json `files` are properly edited. adding mediasoup as a git+ssh dependency of a 3rd party app has also been tested.